cob_description/urdf/cob3_head/head_prosilicas_kinect.urdf.xacro

```diff
@@ -80,7 +80,7 @@
     </xacro:cob_prosilica_v0>
 
     <!-- cam3d -->
-    <xacro:cob_kinect_v0 name="${name}_cam3d" ros_topic="cam3d" parent="${name}_cam_reference_link">
+    <xacro:cob_kinect_v0 name="cam3d" ros_topic="cam3d" parent="${name}_cam_reference_link">
```

Comment:
I revert my comment: that's actually the same as for all cob3 heads. So that's OK